src/Symfony/Component/PropertyInfo/Tests/Extractor/PhpStanExtractorTest.php

```diff
@@ -927,7 +928,11 @@ public static function unionTypesProvider(): iterable
             Type::object(ParentDummy::class),
             Type::null(),
         )];
-        yield ['f', null];
+        if (class_exists(NullableType::class)) { // <- this class was introduced in 7.2
```

Comment:
You can remove this comment, or at least update it a bit, it was to explain my point to you 🙂